spec/index.html

```diff
@@ -259,28 +255,100 @@ <h2>A Canonical form of N-Quads</h2>
       and <code>graphLabel</code> if present, MUST be a single space, 
```

Comment:
```suggestion
      and <code>graphLabel</code> MUST be a single space, 
```
It can not be optional if the goal is a unique serialization (as required by RCH).

Reply:
I don’t believe that “if present” implies optional. What is used for the default graph?

Reply:
"The white space .. if present MUST be a single space, (U+0020)", use of a comma not withstanding.

I think it is better to reword say more clearly that a space must be included.

Reply:
New wording for this point is:

<li>White space MUST NOT be used except after
      <code>subject</code>,
      <code>predicate</code>, 
      <code>object</code>,
      and <code>graphLabel</code>
      where it MUST be a single space (<code>U+0020</code>).</li>